src/components/StorePage/styles/Checkout.css

```diff
@@ -0,0 +1,92 @@
+.checkoutImage {
+  width: 100%;
+}
+
+.checkoutDescription {
+  font-family: 'ArialRoundedMTBold';
+  font-size: 38px;
+  margin-bottom: 0;
+}
+
+.sizeContainer {
+  position: relative;
+  width: 100%;
+  margin: 15% 0;
```

Comment:
Same thing as with the font, I see you used the same margin for `.quantityContainer`. Consider doing 

`.sizeContainer, .quantityContainer {
    margin: 15% 0%;
}` 

also make sure the units you use are consistent! Just give that 0 a percent sign.